eos_data_distribution/producers/usb.py

```diff
@@ -107,7 +107,6 @@ def main():
     GLib.unix_signal_add(GLib.PRIORITY_DEFAULT, signal.SIGTERM, signal_cb)
     monitor = Gio.VolumeMonitor.get()
     store = SimpleStoreProducer(prefix=SUBSCRIPTIONS_SOMA,
-                                split=defaults.ENDLESS_NDN_CACHE_PATH,
```

Comment:
The demo isn't focused on USB. I probably broke it to get other stuff to work. I didn't understand the need for both split and base, so I made everything just use base in there. We can do the same for the USBProducer too.